keyboards/tiger910/readme.md

```diff
@@ -0,0 +1,15 @@
+# Tiger910
+
+![Tiger910](https://www.hualigs.cn/image/60272570d2764.jpg)
+
+A 65% keyboard 
+
+* Keyboard Maintainer:  [Weirdo](https://weirdo-f.github.io)(https://github.com/1248314361)
+* Hardware Supported: Tiger910
+* Hardware Availability: Not yet
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Please add instructions on how to jump into the bootloader to flash new firmware.

Reply:
Bootloader has been modified to Catalina. Do I need to add jump instructions?

Reply:
Yes.  Specifically, the physical "how".  As in, is there a hardware button?  is bootmagic enabled?  location for RESET keycode.  Etc.

Reply:
ok，I have made changes.